### PR TITLE
Implement return functionality

### LINKED ISSUE #5: As a forgetful librarian, I want to mark a book as returned
so that I can let borrowers know that the book will now be available

### LINKED ISSUE #8: As a meticulous libriarian, I want to record the movement of books in and out
so that I can keep track of available books here

### LINKED ISSUE #54: Implement Return command and functionality

### PR DESCRIPTION
**Pull request by seeziyang**
Closes #5 
Closes #8 
Closes #54

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/ReturnCommand.java [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+/**
+ * Returns a Book with the given Index.
+ */
+public class ReturnCommand extends Command {
+    public static final String COMMAND_WORD = "return";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Loans a book to a borrower.\n"
+            + "Command can only be used in Serve mode.\n"
+            + "Parameters: INDEX (must be a positive integer)\n"
+            + "Example: " + COMMAND_WORD + " 1 ";
```
Comment (linyutinglyt) [inline src/main/java/seedu/address/logic/commands/ReturnCommand.java:28]: clear instruction!

File: src/main/java/seedu/address/model/borrower/Borrower.java [hunk 2]
```diff
@@ -16,13 +16,10 @@
...
     private LoanList currentLoanList;
+    private LoanList returnedLoanList;
 
     public Borrower(Name name, Phone phone, Email email) {
-        this.name = name;
-        this.phone = phone;
-        this.email = email;
-        this.borrowerId = BorrowerIdGenerator.generateBorrowerId();
-        this.currentLoanList = new LoanList();
+        this(name, phone, email, BorrowerIdGenerator.generateBorrowerId());
```
Comment (linyutinglyt) [inline src/main/java/seedu/address/model/borrower/Borrower.java:22]: good use of DRY concept!

**Review by linyutinglyt — COMMENTED** [review]
LGTM!

**Review by hoholyin — APPROVED** [review]
LGTM

**Pull request merged**